### ISSUE TITLE
Scan files in long paths

### ISSUE THREAD

**Issue by edsonsbj** [LibrePhotos/librephotos#507]
**Describe the enhancement you'd like**
It would be nice to see in librephotos the possibility for the program to be able to perform the scan in long paths like: 
```
/librephotos/usr/Camera/January
```

**Describe why this will benefit the LibrePhotos**
With this option people running Librephotos together with Nextcloud on the same machine could point librephotos to scan the folder

```
/Nextcloud/data/usr/files/Images
```

Similar to what happens with installing PLEX emby and Jellyfin you can point your media folder to the same folder as independent Nextcloud if the path is too long.

**Comment by derneuere** [comment]
This should work now, because I implemented #136

**Comment by edsonsbj** [comment]
> This should work now, because I implemented #136

thanks friend i just updated and this problem was solved
